src/carbonio-files-ui-common/views/components/List.preview.test.tsx

```diff
@@ -82,23 +82,37 @@ describe('Preview action', () => {
 			'application/vnd.openxmlformats-officedocument.presentationml.presentation';
 		// create XLSX file
 		files[3].mime_type = 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet';
+		// create PDF file
+		files[4].mime_type = 'application/pdf';
 
 		const { user } = setup(<List nodes={files} mainList emptyListMessage="empty list" />);
 
 		await screen.findByText(files[0].name);
 		await screen.findByText(files[1].name);
 		fireEvent.contextMenu(screen.getByText(files[0].name));
 		expect(screen.queryByText(ACTION_REGEXP.preview)).not.toBeInTheDocument();
+
 		fireEvent.contextMenu(screen.getByText(files[1].name));
 		expect(await screen.findByText(ACTION_REGEXP.preview)).toBeVisible();
+		await user.click(screen.getByText(ACTION_REGEXP.preview));
+		// check if there is edit icon in ODT preview
+		expect(screen.getByTestId(ICON_REGEXP.rename)).toBeVisible();
```

Comment:
there is a specific ICON_REGEXP.edit icon for the edit action, use that one instead, since they are created to make the code clearer on what the assertion is checking. Here we are searching for the edit action, but from the code the first thought is that we are searching for the rename.